index.js

```diff
@@ -448,8 +450,11 @@ const vm = new Vue({
 			})
 
 			//only keep 16 color profiles
+			//if (this.colorProfilesMainColors.length > this.MAX_ALT_PALETTES)
+			//	this.colorProfilesMainColors.splice(this.MAX_ALT_PALETTES, this.colorProfilesMainColors.length - 1)
+

```

Comment:
Could you just straight up remove the old code instead of commenting it? It's tracked in git anyway so it's not lost, and it makes it easier to see why stuff is here